mapreader/classify/load_annotations.py

```diff
@@ -109,6 +124,8 @@ def load(
         else:
             self.annotations = annotations
 
+        self._check_patch_paths(remove_broken=remove_broken)
```

Comment:
Looking at this again and the changes to the tests, I think that the assumption here is that all of the paths are valid or that they are all broken. In many cases that is probably true. But if a user manually updates the annotations.csv file, there is a risk that there could be a mixture of broken and valid paths. We should warn them in this case. Writing out the 'broken_files.txt' contains all of the useful information, but does attract the user's attention enough.

Reply:
Current behaviour is to: 
- Check all paths in the .csv file
- Append broken paths to 'broken_files.txt' and remove annotations with broken paths if 'remove_broken=True' (default behaviour). This would leave only valid annotations in place.
- Error if there are no annotations remaining (i.e. all are broken).

Only if the user manually sets 'remove_broken=False' would there be a mixture of broken/valid file paths. This would result in unknown behaviour.

I think your first comment was correct and potentially it would be better for 'remove_broken=False' to result in the third option you suggest: error unless all broken paths are fixed.
I will update this.